tests/lib/rules/max-lines.js

```diff
@@ -21,11 +21,13 @@ const ruleTester = new RuleTester();
 
 /**
  * Returns the error message with the specified max number of lines
- * @param {number} lines Maximum number of lines
+ * @param {number} limitLines Maximum number of lines
+ * @param {number} actualLines Maximum number of lines
```

Comment:
could you update the description of this param? Should be "Actual number of lines" I think